lib/v2/netease/ds.js

```diff
@@ -0,0 +1,36 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const root_url = 'https://inf.ds.163.com';
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    if (!id) {
+        throw Error('Bad parameter. See <a href="https://docs.rsshub.app/game.html#wang-yi-da-shen">https://docs.rsshub.app/game.html#wang-yi-da-shen</a>');
+    }
```

Comment:
Unreachable condition because 
https://github.com/DIYgod/RSSHub/blob/1e22bc7a5240e00539d4662663ca1b97b40b3324/lib/v2/netease/router.js#L6
will throw if `id` does not exist

Reply:
Yeah, this code is copied from the older one that looks like not necessary, I will remove it.